index.css

```diff
@@ -0,0 +1,99 @@
+body {
+  margin: 0;
+  background: url('./images/header-illsutration-mobile.png') no-repeat center 2.5rem;
+  font-family: 'Inter', sans-serif;
+}
+
+ul {
+  list-style-type: none;
+}
+
+li {
+  font-family: 'Inter', serif;
+  font-weight: 600;
+  font-size: 1rem;
+  color: #344563;
+  display: inline-block;
+}
+
+#nav-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.welcome {
+  font-family: 'Inter', serif;
+  font-weight: 600;
+  font-size: 1.25rem;
+  margin-left: 1.5rem;
+}
+
+#mobile-nav {
+  align-self: center;
+  margin-right: 2.5rem;
+}
+
+#headline-section {
+  display: flex;
+  justify-content: center;
+  flex-flow: column;
+  align-items: center;
+  margin-top: 1.5rem;
+}
+
+aside {
+  align-self: center;
+  text-align: center;
+  margin: 0;
+  font-size: 2.5rem;
+}
+
+aside > h1 {
+  color: #172b4d;
+  margin-block: 0;
+  font-weight: bold;
+  line-height: 2.5rem;
+}
+
+aside > p {
+  font-size: 1.44rem;
+  color: #172b4d;
+  margin-block-start: 0.5rem;
+  margin-block-end: 0;
+}
+
+#info {
+  color: #36b37e;
+  display: block;
+}
+
+/* @media only screen and (max-width: 600px) {
+  #desktop-nav {
+    display: none;
+  }
+
+  #mobile-nav {
+    display: flex;
+    justify-content: space-around;
+  }
+
+  #nav-container {
+    display: block;
+  }
+
+  .welcome {
+    font-family: 'inter', serif;
+    font-weight: 600;
+    font-size: 1.25rem;
+  }
+
+  aside > p {
+    font-size: 1.44rem;
+    color: #172b4d;
+    max-width: 100vw;
+  }
+
+  body {
+    background: url('./images/header-illsutration-mobile.png') no-repeat center center;
+  }
+} */
```

Comment:
Please do not leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.